backend/modules/metadata_store/local.py

```diff
@@ -106,8 +106,14 @@ def get_data_source_from_fqn(self, fqn: str = None) -> DataSource | None:
     def get_data_sources(self) -> List[DataSource]:
         return [self.data_source]
 
-    def list_data_sources(self) -> List[str]:
-        return [self.data_source.fqn]
+    async def list_data_sources(self) -> List[str]:
+        return [
+            {
+                "type": self.data_source.fqn.split("::")[0],
+                "uri": self.data_source.fqn.split("::")[1],
```

Comment:
Why do we need to split like this?
Do we not have type and uri separately on the data_source instance itself?

Reply:
Right!, Thanks for bringing it to notice